readme.md

```diff
@@ -80,40 +95,46 @@ Likewise when we write queries, we use **Verbs** to describe the specific data
 
 With the help of Active Record, we can begin to write programs that follow this
 simple pattern to manipulate data.
 
-### Convention Over configuration (ST-WG - 10 / 35)
+### Convention Over configuration (10 / 35)
 
-Before we get started with code, I want to highlight a reoccurring theme with Active Record and Rails in general. You'll often here us say Convention over Configuration.
+Before we get started with code, I want to highlight a reoccurring theme with Active Record and Rails in general. You'll often hear us say, "Convention Over Configuration." Before we discuss the concept as a class, take 30 seconds to think about what that phrase means--why *might* we prefer convention over configuration?
 
 <details>
 <summary>**Question:**  Without getting into the specifics of AR, what do you think we mean by convention over configuration?</summary>
 <br>
 
 ```
-Basically Active Record and Rails, and other frameworks have a whole bunch of conventions that they follow so that you do not have to mess with different configuration details later. These conventions exist because developers agree on best practices, and therefore allows us to spend less time trying to configure, when there all ready is an accepted way to do things. Some of the common ones we will encounter are naming conventions such as: plural vs single, capital or lower, camel or snake.
+ActiveRecord and Rails, and other frameworks have a whole bunch of conventions that they follow so that you do not have to mess with different configuration details later. These conventions exist because developers arrive at a degree of consensus on best practices. These road-tested conventions therefore allow us to spend less time trying to configure, when there already is an accepted way to do things. We inherit from the programmers who have come before us a well-designed, default configuration that spares us from many headaches that we'd encounter if we were building things from scratch (yikes!). Some of the common ones we will encounter are naming conventions such as: plural vs single, Capitalized/ALL_CAPS_SNAKE_CASE/lowercase, camelCase/kabob-case/snake_case. Obeying the naming conventions in ActiveRecord, particularly regarding what is singular vs. what is plural, saves you a good deal of headaches.
 ```
 </details>
 
 
 **BOARD:** Throughout this lesson, I will write on the board Active Record's conventions so we can list them as we go.
 
-In a nutshell, if you don't follow the conventions, you're going to have a bad time.
+##### In a nutshell, if you don't follow the conventions, you're going to have a bad time.
 
-Alright! Let's get started with some code!
+![pizza'd when should have french-fried](http://i62.tinypic.com/2vt47dc.jpg)
+
+Obeying the naming conventions in ActiveRecord, particularly regarding what is singular vs. what is plural, saves you a good deal of headaches.
+
+##### Alright! Let's get started with some code!
 
 ### Setup SQL - Tunr (You Do - 10 / 45)
```

Comment:
Write these headers so that it's clear it's a You Do. So something like...

### You Do: Setup SQL - Tunr (10 / 45)